dot/network/service.go

```diff
@@ -253,6 +256,7 @@ func (s *Service) Start() error {
 	}
 
 	s.host.registerStreamHandler(s.host.protocolID+SyncID, s.handleSyncStream)
+	s.host.registerStreamHandler(s.host.protocolID+WarpSyncID, s.handleWarpSyncStream)
```

Comment:
Can we use the up-to-date protocol name `/{genesis_hash}/sync/warp`? the `{s.host.protocolID}/sync/warp` is a legacy protocol name (that still being used)

https://github.com/paritytech/polkadot-sdk/blob/739c37bfd6df30fac0ffb9b491ee2495e1753054/substrate/client/network/sync/src/warp_request_handler.rs#L64

Reply:
What about [now](https://github.com/ChainSafe/gossamer/pull/4186/commits/f3258f165df447fba351959d38bba16c5b96cf9a)? feel free to resolve this conversation if the solution is ok 😄